vagrant/tutorial/bootstrap.py

```diff
@@ -67,6 +68,16 @@
 # the machine will be reset.
 check_call(['systemctl', 'enable', 'docker'])
 
+
+# Enable firewalld
```

Comment:
I assume the motivation is to make vagrant environment more realistic? In any case, add explanatory comment explaining why we're enabling a firewall.